js/apps/admin-ui/src/realm-settings/user-profile/attribute/AttributeGeneralSettings.tsx

```diff
@@ -135,6 +135,7 @@ export const AttributeGeneralSettings = () => {
           render={({ field }) => (
             <Select
               toggleId="kc-attributeGroup"
+              aria-label={t("attributeGroup")}
```

Comment:
Looks like this one is redundant, as there is already a labelled `FormGroup`.  It seems the `id` is [just mismatched](https://github.com/keycloak/keycloak/blob/160299d3c64e3543c08470d01fa5186025bfd010/js/apps/admin-ui/src/realm-settings/user-profile/attribute/AttributeGeneralSettings.tsx#L129).

Reply:
Thanks @jonkoops This is a strange one. I have now both fieldId and toggleId the same, but if I remove the aria-label I get the violation. It seems that the aria-label is needed here.

Reply:
Very strange, perhaps the ID is not being applied correctly?

Reply:
Upon further inspection, this is a bug in PatternFly, looks like passing the `aria-label` attribute is the only way [this functions](https://github.com/patternfly/patternfly-react/blob/db0cdde6e3e6ae61348129b73bb43c97a98b4617/packages/react-core/src/components/Select/SelectMenu.tsx#L202).